pkg/speaker/bgp/bgp/conf.go

```diff
@@ -28,7 +29,7 @@ func (b *Bgp) HandleBgpGlobalConfig(global *bgpapi.BgpConf, rack string, delete
 	}
 	err = b.updatePolicy(cm)
 	if err != nil {
```

Comment:
Is this error not required to be returned?

Reply:
> Is this error not required to be returned?

We should return this error.